### PR TITLE
Remove multi-arch from older platforms

### PR DESCRIPTION
**Pull request by timja**
Build is currently taking a bit long (https://github.com/jenkinsci/docker/issues/1139#issuecomment-885664122), let's see if this helps

cc @dduportal

### CODE REVIEW

**Review by dduportal — APPROVED** [review]
🚀 LGTM

**Review by slide — APPROVED** [review]

File: docker-bake.hcl [hunk 1]
```diff
@@ -1,8 +1,6 @@
 group "linux" {
   targets = [
     "almalinux_jdk11",
-    "alpine_jdk8",
-    "centos7_jdk8",
```
Comment (MarkEWaite) [inline docker-bake.hcl:5]: I don't understand these removals.  Doesn't removing these two mean that we will stop delivering an Alpine Linux image that uses JDK 8?

Comment (timja) [reply docker-bake.hcl]: oops, those weren't meant to be committed

**Review by MarkEWaite — APPROVED** [review]
Looks great.  Makes the story clear that new platforms will only get Java 11.

**Pull request merged**